### ISSUE TITLE
ElectrumABC issue ledger

### ISSUE THREAD

**Issue by dokflide** [Bitcoin-ABC/ElectrumABC#199]
After updating electrumABC and apps on ledger live, ElectrumABC doesn't recognise any more my ledger.

**Comment by PiRK** [comment]
I'm investigating. 
After upgrading apps and firmware in Ledger MAnager, my Ledger Nano X no longer working.

I'm getting the following error:
```

Traceback (most recent call last):
  File "/home/pierre/dev/ElectrumABC/electroncash_plugins/hw_wallet/qt.py", line 254, in run
    result = task.task()
  File "/home/pierre/dev/ElectrumABC/electroncash_plugins/ledger/ledger.py", line 685, in get_client
    client = devmgr.client_for_keystore(self, handler, keystore, force_pair)
  File "/home/pierre/dev/ElectrumABC/electroncash/plugins.py", line 879, in client_for_keystore
    client = self.force_pair_xpub(plugin, handler, info, xpub, derivation)
  File "/home/pierre/dev/ElectrumABC/electroncash/plugins.py", line 921, in force_pair_xpub
    client_xpub = client.get_xpub(derivation, xtype)
  File "/home/pierre/dev/ElectrumABC/electroncash_plugins/ledger/ledger.py", line 52, in catch_exception
    return func(self, *args, **kwargs)
  File "/home/pierre/dev/ElectrumABC/electroncash_plugins/ledger/ledger.py", line 106, in get_xpub
    self.checkDevice()
  File "/home/pierre/dev/ElectrumABC/electroncash_plugins/ledger/ledger.py", line 223, in checkDevice
    self.perform_hw1_preflight()
  File "/home/pierre/dev/ElectrumABC/electroncash_plugins/ledger/ledger.py", line 172, in perform_hw1_preflight
    firmwareInfo = self.dongleObject.getFirmwareVersion()
  File "/home/pierre/.local/lib/python3.10/site-packages/btchip/btchip.py", line 563, in getFirmwareVersion
    response = self.dongle.exchange(bytearray(apdu))
  File "/home/pierre/.local/lib/python3.10/site-packages/btchip/btchipComm.py", line 90, in exchange
    result = self.waitImpl.waitFirstResponse(timeout)
  File "/home/pierre/.local/lib/python3.10/site-packages/btchip/btchipComm.py", line 134, in waitFirstResponse
    data = self.device.read(65)
  File "hid.pyx", line 219, in hid.device.read
OSError: read error

```

**Comment by haplo** [comment]
Upstream backwards-incompatible change?

https://github.com/LedgerHQ/btchip-python/issues/52

**Comment by PiRK** [comment]
There is a (temporary?) fix in the 2.0.5 app https://github.com/LedgerHQ/app-bitcoin-new/pull/36

**Comment by dokflide** [comment]
Can you explain please how

Op vr 20 mei 2022 13:27 schreef Pierre K ***@***.***>:

**Comment by PiRK** [comment]
I think we will have to wait until the new version of the app is available in Ledger Live, and then upgrade it. Yesterday it wasn't available, and I didn't check today.

**Comment by dokflide** [comment]
Ok, thanks for the update.

Op vr 20 mei 2022 17:37 schreef Pierre K ***@***.***>:

**Comment by PiRK** [comment]
A new version of the BCH app is available in Ledger Live (Ledger Manager) : 2.0.5
For me it solves the issue.
